src/api/java/com/minecolonies/api/client/render/modeltype/registry/IModelTypeRegistry.java

```diff
@@ -1,23 +1,32 @@
 package com.minecolonies.api.client.render.modeltype.registry;
 
 import com.minecolonies.api.IMinecoloniesAPI;
-import com.minecolonies.api.client.render.modeltype.CitizenModel;
 import com.minecolonies.api.client.render.modeltype.IModelType;
-import com.minecolonies.api.entity.citizen.AbstractEntityCitizen;
-
-import java.util.Map;
+import net.minecraft.util.ResourceLocation;
 
+/**
+ *
+ */
 public interface IModelTypeRegistry
 {
-
+    /**
+     * @return
+     */
     static IModelTypeRegistry getInstance()
     {
         return IMinecoloniesAPI.getInstance().getModelTypeRegistry();
     }
 
-    IModelTypeRegistry register(IModelType type, CitizenModel<AbstractEntityCitizen> maleModel, CitizenModel<AbstractEntityCitizen> femaleModel);
-
-    Map<IModelType, CitizenModel<AbstractEntityCitizen>> getMaleMap();
+    /**
+     * @param name
+     * @param modelType
+     * @return
+     */
+    void register(IModelType modelType);
```

Comment:
See this is way better.
If you have a registry, why are you exposing the implemented object then?